src/Compilers/Test/Utilities/VisualBasic/BasicTestBase.vb

```diff
@@ -176,7 +176,7 @@ Public MustInherit Class BasicTestBase
 
         Dim assemblyName As String = Nothing
         Dim sourceTrees = ParseSourceXml(source, parseOptions, assemblyName)
-        Dim compilation = CreateEmptyCompilation(sourceTrees.ToArray(), allReferences, options, assemblyName:=assemblyName)
+                        Dim compilation = CreateEmptyCompilation(sourceTrees.ToArray(), allReferences, options, parseOptions, assemblyName:=assemblyName)
```

Comment:
please don't change formatting unnecessarily

Reply:
It the online editor cause the formatting change, I go an fix it.